sklearn/tests/test_cross_validation.py

```diff
@@ -95,7 +95,7 @@ def fit(self, X, Y=None, sample_weight=None, class_prior=None,
     def predict(self, T):
         if self.allow_nd:
             T = T.reshape(len(T), -1)
-        return T.shape[0]
+        return T[:, 0]
```

Comment:
As this is not part of the public API (mock utils part of a test file) I don't see any problem for changing this one.